js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js

```diff
@@ -15,7 +15,7 @@
 
       // rerender
       self.interval = window.setInterval(function () {
-        if (window.location.hash.indexOf('cIndices/' + self.collectionName) !== -1 && window.VISIBLE) {
+        if (window.location.hash.indexOf('cIndices/' + encodeURIComponent(self.collectionName)) !== -1 && window.VISIBLE) {
```

Comment:
As this handles URLs where the hash part looks like this, there's probably a link somewhere in a template that needs to be updated alongside this change.

Reply:
The `cIndices` routes appears only here, so I had hoped we are covered:
```
$ grep -r "cIndices" js/apps/system/_admin/aardvark/APP/frontend/js/
js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js:          route: '#cIndices/' + encodeURIComponent(collectionName)
js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js:        if (window.location.hash.indexOf('cIndices/' + encodeURIComponent(self.collectionName)) !== -1 && window.VISIBLE) {
js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js:      'cIndices/:colname': 'cIndices',
js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js:    cIndices: function (colname) {
```

Reply:
```
js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js:    cIndices: function (colname) {
```
Note that `colname` will now be the escaped name. The function may need to unescape it before handling it.

Reply:
I don't think so. I just verified it myself that the parameters that the Backbone router passes into the registered functions are already URL-decoded. So I think it is fine.